styles/style.css

```diff
@@ -345,7 +346,220 @@ h1 {
 
 /* Minimum desktop screen size should be 768px. this will be changed in the next summits when i make the deskopt. */
 @media screen and (min-width: 768px) {
-  body {
-    background-color: black;
+  .toolbar {
+    padding: 0.1rem 9.6rem 0.1rem 9.6rem;
+  }
+
+  .menu {
+    display: none;
+  }
+
+  .toolbar div {
+    display: flex;
+    align-items: center;
+  }
+
+  .navigation {
+    display: flex;
+    justify-content: space-between;
+  }
+
+  .navigation a {
+    margin: 30px;
+    text-decoration: none;
+  }
+
+  .headline {
+    background-color: #fff;
+    display: flex;
+    flex-direction: column;
+    align-items: start;
+    padding: 300px 260px 200px 260px;
+    border-bottom-left-radius: 80px;
+  }
```

Comment:
- [ ] Please adjust the font-weight of your main title and description paragraph in both desktop and mobile version to make it look like figma

**_Figma Design_**

<img width="361" alt="Screenshot 2023-02-24 at 12 03 34 AM" src="https://user-images.githubusercontent.com/44798044/221005352-40446f9a-8797-420a-8a7d-53d9a07f72d2.png">

<img width="133" alt="Screenshot 2023-02-24 at 12 03 44 AM" src="https://user-images.githubusercontent.com/44798044/221005380-ea50bd7f-ac2f-473c-bc52-42e2ec98a8cf.png">

Your design

<img width="605" alt="Screenshot 2023-02-24 at 12 04 44 AM" src="https://user-images.githubusercontent.com/44798044/221005637-32312e0d-4104-45a9-a8c9-85ee2fbd17c6.png">

<img width="441" alt="Screenshot 2023-02-24 at 12 04 54 AM" src="https://user-images.githubusercontent.com/44798044/221005566-810d1b36-c3f7-497d-9832-339dae70d804.png">